packages/terra-hookshot/src/terra-dev-site/test/common/HookshotTestTemplate.jsx

```diff
@@ -81,8 +83,32 @@ class HookshotTemplate extends React.Component {
       buttonMarginTop,
       boundingWidth,
       attachmentMargin,
+      includeSvgs,
       ...hookshotProps
     } = this.props;
+
+    const svgs = (
+      <div>
```

Comment:
This is used for manually verifying in IE 10 / IE 11 that clicking on an SVG will close hookshot content.